Directory.Build.props

```diff
@@ -29,7 +29,7 @@
     <CastleCoreVersion>4.4.0</CastleCoreVersion>
     <NSubstituteVersion>3.1.0</NSubstituteVersion>
     <NewtonsoftJsonVersion>12.0.2</NewtonsoftJsonVersion>
-    <GraphQLVersion>3.0.0-preview-1260</GraphQLVersion>
+    <GraphQLVersion>3.0.0-preview-1268</GraphQLVersion>
```

Comment:
I pushed 1268 to nuget.org and I'll push these packages to nuget.org once its merged.

Reply:
Oh, a pleasant surprise. Do you push packages manually, copying them from myget?

Reply:
Yes - I login to myget and push packages from there.  I made this a manual process so that we weren't pushing every small build and branch to Nuget.org.

Reply:
It does look like something got messed up with pushing packages to myget.org with this project.  I'm not sure what yet ...

Reply:
Maybe some outdated API keys? Just guessing.